app/src/pages/LoginPage.vue

```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import LButton from "@/components/button/LButton.vue";
 import LCard from "@/components/common/LCard.vue";
-import BccLogo from "@/components/BccLogo.vue";
+import LLogo from "@/components/LLogo.vue";
```

Comment:
Do not change this yet (keep the original BCC logo). This will be updated when the Auth0 integration is redone